homeassistant/components/qnap/strings.json

```diff
@@ -22,5 +22,54 @@
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "status": {
+        "name": "Status"
+      },
+      "system_temp": {
+        "name": "System temperature"
+      },
+      "cpu_temp": {
+        "name": "CPU temperature"
+      },
+      "cpu_usage": {
+        "name": "CPU usage"
+      },
+      "memory_free": {
+        "name": "Memory available"
+      },
+      "memory_used": {
+        "name": "Memory used"
+      },
+      "memory_percent_used": {
+        "name": "Memory usage"
+      },
+      "network_link_status": {
+        "name": "Network link ({monitor_device})"
+      },
+      "network_tx": {
+        "name": "Network up ({monitor_device})"
+      },
+      "network_rx": {
+        "name": "Network down ({monitor_device})"
+      },
+      "drive_smart_status": {
+        "name": "SMART status (drive {monitor_device})"
+      },
+      "drive_temp": {
+        "name": "Temperature (drive {monitor_device})"
+      },
+      "volume_size_used": {
+        "name": "Used space ({monitor_device})"
+      },
+      "volume_size_free": {
+        "name": "Free space ({monitor_device})"
+      },
+      "volume_percentage_used": {
+        "name": "Volume used ({monitor_device})"
+      }
```

Comment:
Just a thought, let me know what you think of it.

Now we are changing naming, can we also look at how the names look? Can we make it more descriptive:

NAS xxx Network up (eth0) -> NAS xxx eth0 up
NAS xxx Temperature (drive 1) -> NAS xxx drive 1 temperature

Reply:
I tried not to change the names but leave them as they've been before:

<img width="238" alt="Bildschirmfoto 2024-01-05 um 11 57 27" src="https://github.com/home-assistant/core/assets/271149/26b75047-f062-42dd-96bd-76aa2669ca48">

Reply:
I mean that's also fine, but now we are changing names, might as well review them. I also know @disforw tried implementing translations, so maybe he also has an opinion.

Reply:
I was following that placeholder PR closely and saw it was merged yesterday, you guys beat me to it! 😁

@joostlek if you're looking for a unit name or identifier, that's already being brought over using the device name.
![Screenshot_20240105-071535](https://github.com/home-assistant/core/assets/24241104/408ce824-3e49-4716-aa0c-3012ec956834)

Reply:
I know, but my question was more of the stuff behind it. Do we want to change that to make it more fluent or do we keep it like this